### PR TITLE
Fix backend init container sentinel env

### PR DESCRIPTION
**Pull request by alanmoran**

### CODE REVIEW

File: pkg/3scale/amp/component/backend.go [hunk 1]
```diff
@@ -204,6 +204,8 @@ func (backend *Backend) buildBackendWorkerDeploymentConfig() *appsv1.DeploymentC
 						}, Env: []v1.EnvVar{
 							envVarFromValue("SLEEP_SECONDS", "1"),
 							envVarFromSecret("CONFIG_QUEUES_MASTER_NAME", BackendSecretBackendRedisSecretName, BackendSecretBackendRedisQueuesURLFieldName),
+							envVarFromSecret("CONFIG_QUEUES_SENTINEL_HOSTS", BackendSecretBackendRedisSecretName, BackendSecretBackendRedisQueuesSentinelHostsFieldName),
```
Comment (eguzki) [inline pkg/3scale/amp/component/backend.go]: It is better to use backend common env to have all env variables available

Something like this:

```
Env: append(backend.buildBackendCommonEnv(), envVarFromValue("SLEEP_SECONDS", "1")),
```

File: pkg/3scale/amp/component/backend.go [hunk 2]
```diff
@@ -287,6 +289,8 @@ func (backend *Backend) buildBackendCronDeploymentConfig() *appsv1.DeploymentCon
...
```
Comment (eguzki) [inline pkg/3scale/amp/component/backend.go]: Same as above, something like this:
```
Env: append(backend.buildBackendCommonEnv(), envVarFromValue("SLEEP_SECONDS", "1")),
```

File: pkg/3scale/amp/component/backend.go (hunk 1 above)
Comment (alanmoran) [reply pkg/3scale/amp/component/backend.go]: Fixed

File: pkg/3scale/amp/component/backend.go (hunk 2 above)
Comment (alanmoran) [reply pkg/3scale/amp/component/backend.go]: Fixed

**Comment by miguelsorianod** [comment]
Looks good to me

**Pull request closed without merging**